app/build.gradle.kts

```diff
@@ -25,15 +25,13 @@ android {
     }
 
     buildTypes {
-        debug {
-            isMinifyEnabled = false
-        }
         release {
-            isMinifyEnabled = true
+            isMinifyEnabled = false
```

Comment:
Likely this should be reverted as you want minification enabled for the `release` build type.

Reply:
this is just for experimentation